notebooks/234-blip-visual-language-processing/234-blip-visual-language-processing.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>We need to add comments into the code. We would like to avoid devs to debug that a lot. If they can read the code and understand that in a simple way will be better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>